edb/edgeql/tracer.py

```diff
@@ -281,11 +283,59 @@ def trace_refs(
         modaliases={},
         params=params,
         visited=set(),
+        local_modules=local_modules,
     )
     trace(qltree, ctx=ctx)
     return frozenset(ctx.refs)
 
 
+def get_name(
```

Comment:
Rename to `resolve_relative_name` or `lookup_relative_name`?

Reply:
I went with just `resolve_name`